prime-angular-frontend/src/app/modules/enrolment/pages/care-setting/care-setting.component.ts

```diff
@@ -248,11 +248,13 @@ export class CareSettingComponent extends BaseEnrolmentProfilePage implements On
     const form = this.enrolmentFormStateService.oboSitesForm;
     const oboSites = form.get('oboSites') as FormArray;
 
-    oboSites.value?.forEach((site: OboSite, index: number) => {
-      if (site.careSettingCode === careSettingCode) {
-        oboSites.removeAt(index);
+    for (var i = 0; i < oboSites.length;) {
```

Comment:
Could the same be achieved by going through the array from last element to first?

Reply:
I think so. If it works one way, should work in the other way. But, I don't see advantage one way rather than other.

Reply:
It is updated. Thank you for pointing out a way to improve the coding.